src/Messages/OWSIdentityManager.m

```diff
@@ -190,6 +192,14 @@ - (BOOL)saveRemoteIdentity:(NSData *)identityKey recipientId:(NSString *)recipie
             // Cancel any pending verification state sync messages for this recipient.
             [self clearSyncMessageForRecipientId:recipientId];
 
+            // TODO: This may be redundant with the "safety number changes"
+            //       messages.
```

Comment:
I suspect we'll _often but not always_ see these right after the "safety number change" messages. I'm wondering about the best way to avoid duplication without missing corner cases.